### ISSUE TITLE
Gamepad Controlled Support

### ISSUE THREAD

**Issue by christopherghenderson** [aviks/GameZero.jl#65]
I'm curious if theres any plan to put in gamepad support, given its SDL2 it should be relatively doable I assume?

**Comment by aviks** [comment]
Yeah, this should be feasible. I can't promise if I can get to it any time soon, especially since I don't have a gamepad handy.
